build-tests.js

```diff
@@ -0,0 +1,127 @@
+'use strict'
+
+var fs = require('fs')
+var Path = require('path')
```

Comment:
Because I named another variable `path` down below. Normally I'd just rename the other variable, but `path` is the de facto variable name for your node path in AST transforms.

I've extracted the transform into it's own module anyways, so I can fix this up.